### PR TITLE
VZ:9164: Add CAPI verify tests

### PR DESCRIPTION
**Pull request by Abhijithang**
This PR adds cluster api tests to verify install test suite.

### CODE REVIEW

File: pkg/constants/constants.go [hunk 1]
```diff
@@ -49,6 +49,9 @@ const VerrazzanoInstallNamespace = "verrazzano-install"
 // VerrazzanoSystemNamespace is the system namespace for Verrazzano
 const VerrazzanoSystemNamespace = "verrazzano-system"
 
+// VerrazzanoCapiNamespace is the system namespace for Cluster API resources
+const VerrazzanoCapiNamespace = "verrazzano-capi"
```
Comment (mgianatagh) [inline pkg/constants/constants.go:53]: In capi_component.go there is a constant named VerrazzanoCAPINamespace.  Can you change that constant to equal the constant you have defined globally?  

VerrazzanoCAPINamespace = constants.VerrazzanoCapiNamespace

File: tests/e2e/verify-install/capi/capi_suite_test.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package capi
```
Comment (mgianatagh) [inline tests/e2e/verify-install/capi/capi_suite_test.go:1]: Missing copyright header

File: tests/e2e/verify-install/capi/capi_test.go [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+	isCapiSupported, err = pkg.IsVerrazzanoMinVersion("1.6.0", kubeconfigPath)
+	if err != nil {
+		AbortSuite(fmt.Sprintf("Failed to check Verrazzano version 1.6.0: %v", err))
+	}
+	if isCapiSupported {
+		update.UpdateCRV1beta1WithRetries(m, pollingInterval, waitTimeout)
+		update.ValidatePods(capiLabelValue, capiLabelKey, constants.VerrazzanoCapiNamespace, uint32(4), false)
+	}
+	inClusterVZ, err = pkg.GetVerrazzanoInstallResourceInClusterV1beta1(kubeconfigPath)
```
Comment (anders-swanson) [inline tests/e2e/verify-install/capi/capi_test.go:106]: Error is not handled

File: tests/e2e/verify-install/capi/capi_suite_test.go (hunk 2 above)
Comment (Abhijithang) [reply tests/e2e/verify-install/capi/capi_suite_test.go:4]: Thanks for finding it, I fixed it now.

File: tests/e2e/verify-install/capi/capi_test.go (hunk 3 above)
Comment (Abhijithang) [reply tests/e2e/verify-install/capi/capi_test.go:106]: Fixed it, thanks.

**Review by anders-swanson — DISMISSED** [review]

**Pull request merged**